mobile/src/stateMachine/views/password/setPassword.tsx

```diff
@@ -38,21 +36,9 @@ export class SetPasswordView extends PasswordBase {
             return;
         }
 
-        const res = await this.viewModel.updatePassword();
+        const res = await this.viewModel.resetPassword();
 
-        if (res === 'magicLink') {
```

Comment:
I am not sure if this file is using the magic link feature verification

Reply:
I have tested with magic link and it's working.